### DISCUSSION TITLE
Tax certificates and documents from Github for Tax saving on billing amount in India

### DISCUSSION THREAD

**Discussion by CJD-Admin** [community/community#135753] [Copilot Conversations]
### Select Topic Area

General

### Body

I want to get Tax certificate documents on email for Tax saving on billing amount in India.

**Comment by Akash1134** [comment]
Hi @CJD-Admin, 

Receipts are sent automatically upon transaction processing and can be downloaded from your billing history. If needed, you can also[ request a receipt here](https://support.github.com/contact/receipt).

You can add information to your personal account's receipts, such as a VAT or GST identification number, or your full business name or address of record.

1.  In the upper-right corner of any page on GitHub, click your profile photo, then click **Settings**.

2.  In the "Access" section of the sidebar, click **Billing and plans**, then click **Payment information**.

3.  At the bottom of the page, next to "Additional information," click **Add information**.

4.  In the text fields, type the extra information you'd like on your receipt, then click **Save contact information**.

Thanks!
